include/obmc_hypervisor.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+#include <sys/socket.h>
+
+#include <app.hpp>
+#include <async_resp.hpp>
+#include <boost/asio/local/stream_protocol.hpp>
+#include <boost/container/flat_map.hpp>
+#include <boost/container/flat_set.hpp>
+#include <websocket.hpp>
+
```

Comment:
We should include what we use 
include string, array  
#include <memory> (unique_ptr)

Reply:
Done; except for unique_ptr - couldn't figure out the include.